templates/kubernetes-cluster.template

```diff
@@ -977,21 +987,33 @@ Resources:
       - Key: 'kubernetes.io/service-name'
         Value: 'kube-system/apiserver-public'
 
-  # Security group to allow public access to port 443
+  # Security group to allow public access to port 443 and 6443
   ApiLoadBalancerSecGroup:
     Type: AWS::EC2::SecurityGroup
     Properties:
       GroupDescription: Security group for API server load balancer
       VpcId: !Ref VPCID
       SecurityGroupIngress:
-        CidrIp: !Ref ApiLbLocation
-        FromPort: 443
-        ToPort: 443
-        IpProtocol: tcp
+        - CidrIp: !Ref ApiLbLocation
```

Comment:
oh, nice catch with the list...i don't know YAML well enough I guess? Did this cloudformation change or is what we had before shorthand for a single list item?. 

Also, could you unindent this 2 spaces to keep with the style of the rest of the file?

```yaml
likethis:
- item: 1
  in: 2
  list: 3

#vs

notlikethis:
  - item: 1
    in: 2
    list: 3
```

Reply:
Will do, it was a shorthand for a single item list, although found it confusing so added a `-` :D